ToDoLite/DetailViewController.m

```diff
@@ -226,6 +246,18 @@ - (UITableViewCell *)couchTableSource:(CBLUITableSource *)source cellForRowAtInd
                                                     forIndexPath:indexPath];
     CBLQueryRow *row = [source rowAtIndex:indexPath.row];
     Task *task = [Task modelForDocument:row.document];
+
+    int64_t delay = 1.5;
+    // avoid having all text start gray only to immediately turn black (otherwise there is a flicker effect that users might think is a bug)
+    dispatch_time_t time = dispatch_time(DISPATCH_TIME_NOW, delay * NSEC_PER_SEC);
+    dispatch_after(time, dispatch_get_main_queue(), ^(void){
+        if ([repl isDocumentPending:task.document]) {
+            cell.name.textColor = [UIColor grayColor];
+        } else {
+            cell.name.textColor = [UIColor blackColor];
+        }
+    });
+
```

Comment:
I think there could be a side effect of delaying text color setting here when the cells get reused by other rows. Basically each row could randomly show the text in either gray or black and then changes later. Another issue could be that after 1.5 seconds, the color might get set on the wrong cell; for example, the order of the row to set the color gets updated from the live query.

In general, delay updating the row cell is not simple and is required some more work. I'm wondering how much the flicker effect is if we just set the color without delaying. Note that I haven't tried to run the app yet.